.github/ISSUE_TEMPLATE/bug-openapi-typescript.md

```diff
@@ -0,0 +1,56 @@
+---
+name: "Bug report: openapi-typescript"
+about: For the openapi-typescript library
+title: ""
+labels: openapi-ts, bug
+assignees: ""
+body:
+  - type: input
+    attributes:
+      label: openapi-typescript version
+      placeholder: x.x.x
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Node.js version
+      placeholder: 20.x.x
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: OS + version
+      placeholder: macOS 15.1.1
+    validations:
+      required: true
```

Comment:
Do we see OS specific issues a lot? (then a gain, it's little to ask, so maybe better to ask right away).

Reply:
Especially with openapi-typescript, quite a lot! It’s also why we have Windows tests. Anything that talks to the filesystem, really, it’s important to know what OS they’re on. Put another way, it’s basically a “are you using Windows” question.

Reply:
> Anything that talks to the filesystem, really, it’s important to know what OS they’re on.

Ah, yes, good point!